Library/Homebrew/tap.rb

```diff
@@ -62,6 +62,7 @@ def initialize(user, repo)
     @repo = repo
     @name = "#{@user}/#{@repo}".downcase
     @path = TAP_DIRECTORY/"#{@user}/homebrew-#{@repo}".downcase
+    @remote = remote
```

Comment:
You should use a different name for this variable, like requested_remote or something.